msg/vehicle_control_mode.msg

```diff
@@ -12,6 +12,7 @@ bool flag_system_hil_enabled
 bool flag_control_manual_enabled		# true if manual input is mixed in
 bool flag_control_auto_enabled			# true if onboard autopilot should act
 bool flag_control_offboard_enabled		# true if offboard control should be used
+bool flag_control_offboard_actuator_control_enabled #true if raw actuator control in offboard used
```

Comment:
as discussed per email, this should not be necessary. E.g flag_control_rates = flag_control_attitude = false (etc.) will disable the attitude controller. Then you don't need to change the attitude controller.